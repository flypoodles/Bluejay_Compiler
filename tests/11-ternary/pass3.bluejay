fun recursive void (i int)
  var n int := i-1;

  if (i == 0)  return;
  printint(i);
  printstring(" ");
  printstring((i%2) ? "odd" : "even");
  printstring(" ");
  recursive(n);


recursive(10);
return 0;
