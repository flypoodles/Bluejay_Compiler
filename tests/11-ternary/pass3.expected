10 even 9 odd 8 even 7 odd 6 even 5 odd 4 even 3 odd 2 even 1 odd 
